cmd/create/admin/cmd.go

```diff
@@ -111,25 +107,32 @@ func run(cmd *cobra.Command, _ []string) {
 
 	// No HTPasswd IDP - create it with cluster-admin user.
 	if existingHTPasswdIDP == nil {
-		r.Reporter.Debugf("Adding '%s' idp to cluster '%s'", idpName, clusterKey)
+		r.Reporter.Debugf("Adding '%s' idp to cluster '%s'", idp.HTPasswdIDPName, clusterKey)
 		htpasswdIDP := cmv1.NewHTPasswdIdentityProvider().Users(cmv1.NewHTPasswdUserList().Items(
 			idp.CreateHTPasswdUser(idp.ClusterAdminUsername, password),
 		))
 		newIDP, err := cmv1.NewIdentityProvider().
 			Type("HTPasswdIdentityProvider").
-			Name(idpName).
+			Name(idp.HTPasswdIDPName).
 			Htpasswd(htpasswdIDP).
 			Build()
 		if err != nil {
-			r.Reporter.Errorf("Failed to create '%s' identity provider for cluster '%s'", idpName, clusterKey)
+			r.Reporter.Errorf("Failed to create '%s' identity provider for cluster '%s'", idp.HTPasswdIDPName, clusterKey)
 			os.Exit(1)
 		}
 
 		// Add HTPasswd IDP to cluster:
 		_, err = r.OCMClient.CreateIdentityProvider(cluster.ID(), newIDP)
 		if err != nil {
+			//since we could not add the HTPasswd IDP to the cluster, roll back and remove the cluster admin
 			r.Reporter.Errorf("Failed to add '%s' identity provider to cluster '%s': %s",
```

Comment:
Failed to add '%s' identity provider to cluster as part of admin flow.  Please try again.